windows-10-setup.md

```diff
@@ -7,45 +10,132 @@
 1. Install Ubuntu 20 and the Windows Terminal from the Windows Store
     - If you can't install from the Windows Store, make sure you're signing in with your Microsoft Account and have verified your device (Settings -> Accounts)
     - Also, make sure you're using the most recent version of Windows 10. On the Windows Terminal page in the Store, select System Requirements -> Update
-    - Open Ubuntu and let it finish installation. If it says installing for more than 5 minutes, close the Ubuntu window and re-open it
+    - Open Ubuntu and let it finish installation
+        - When prompted, enter a username (e.g. your first name) and a password - this is the username that Linux will run as by default
+        - For Full Name, Room Number, etc. you can hit Enter to leave them blank
+        - If it says installing for more than 5 minutes, close the Ubuntu window and re-open it
 1. Configure your Windows Terminal
-    - Select the dropdown next to the new tab button and select Settings
-    - In the file change the value of the `defaultProfile` property to the value of the Ubuntu's `guid` property _(which you should see lower down in the file)_
-    - Add these properties to the Ubuntu section 
+    - Open your Windows Terminal application and select the dropdown next to the new tab button then select Settings
+    - Scroll down until you find this section:
+ 
+    ```js
+       {
+                "guid": "{07b52e3e-de2c-5db4-bd2d-ba144ed6c273}",
+                "hidden": false,
+                "name": "Ubuntu-20.04",
+                "source": "Windows.Terminal.Wsl"
+       },
+    ```
+
+    - Copy the value of the "guid" field, then paste it in to the `defaultProfile` property near the top of the file.
+    
+    ```js
+        "defaultProfile": "{07b52e3e-de2c-5db4-bd2d-ba144ed6c273}",
+    ```
+    
+    - Next, add these properties to the Ubuntu section (the same section you got the "guid" from in the previous step) 
     ```js
     "colorScheme": "One Half Dark",
-    "startingDirectory": "//wsl$/Ubuntu-20.04/home/YOUR_USERNAME"
+    "startingDirectory": "//wsl$/Ubuntu-20.04/home/[your_Linux_username]"
```

Comment:
I've noticed that all of the Windows users I've been helping during this (and the previous) bootcamp have all been the `root` user in Linux. I'm not sure how they did this. Did they not select a username during the Ubuntu install? I'm not sure. So the value of `startingDirectory` has been `//wsl$/Ubuntu-20.04/home/root`. I need to get to the bottom of this, but I don't currently have a Windows machine I can test this on. Have you noticed this @josephquested?

Reply:
I recall this happened to me when I was installing WSL. When it got to the question about "Room Number" or something I got spooked and tried to quit out of the install, but by that point I think it had done the install + root bit but not created my user account. And then Google + Joseph helped me to add a user account, so my install is fine now. Clearly something similar is still happening for students though. Some months back I did add some expanded tips about the Room Number etc. in this guide (Joseph's version only, until now) but obviously something is still going wrong for students so like you say Don, it would be good to know how/why.